src/core/js/a11y/browserFocus.js

```diff
@@ -62,7 +62,9 @@ export default class BrowserFocus extends Backbone.Controller {
    */
   _onClick(event) {
     const $element = $(event.target);
-    if ($element.is('[aria-disabled=true]')) {
+    // search elemenent and parents for aria-disabled - see https://github.com/adaptlearning/adapt_framework/issues/3097
+    const isAriaDisabled = $element.closest('[aria-disabled=true]') > 0;
```

Comment:
This will never resolve to a truthy.

`$element.closest('[aria-disabled=true]') > 0` = `false`
`$element.closest('[aria-disabled=true]').length` = `0` | `1`
`$element.closest('[aria-disabled=true]').length !== 0` = `true` | `false`
`Boolean($element.closest('[aria-disabled=true]').length)` = `true` | `false`

Reply:
`$element.closest('[aria-disabled=true]')` is a jQuery object, needs `.length` on the end.

Reply:
oops - thanks both!